notebooks/test_notebook/test_notebook.py

```diff
@@ -0,0 +1,76 @@
+# test_input_unittest.py
```

Comment:
Please put this into a directory called `tests` (not `test_notebook` and name it something more informative, like `test_tellseq_D_variable_volume_pooling.py`--we have half a dozen notebooks that will all need (separate) testing modules.